cicerone/app/src/main/java/com/teampower/cicerone/GeofenceTriggeredActivity.kt

```diff
@@ -146,11 +149,23 @@ class GeofenceTriggeredActivity : AppCompatActivity() {
         favorite_button.setOnClickListener { toggleFavoritePOI(POI) }
         Log.v(TRIG_TAG, POI.toString())
 
-        // Demo on how to update category score
+        // User-feedback for recommendation
         catViewModel = ViewModelProvider(this).get(CategoryViewModel::class.java)
-        catViewModel.updateCategoryPoints("School", 2.0) // Change score
+        like_button.setOnClickListener {
+            catViewModel.like(POI.categoryID)
+            like_button.isEnabled = false
+            dislike_button.isEnabled = false
+        }
+
+        dislike_button.setOnClickListener {
+            catViewModel.dislike(POI.categoryID)
+            like_button.isEnabled = false
+            dislike_button.isEnabled = false
+        }
+        // TODO table doesn't seem to update
```

Comment:
In what way doesn't the table seem to update?

Reply:
It is working now, I forgot to remove the comment 😃